lib/account/account-manager.js

```diff
@@ -19,6 +36,28 @@
  * @typedef {Object} accountManager~confirmedInfo
  * @property {string} mfaKey
  * @property {string} passwordSalt
+ * @property {accountManager~hashSet} pbkdf2
+ */
+
+/**
+ * How to hash a password for registering and logging in.
+ *
+ * @typedef {Object} accountManager~hashSet
+ * @property {string} algo
+ * @property {string} bytes
+ * @property {string} iterations
+ * @property {string} saltLength
+ */
+
+/**
+ * What an account holder needs to create the password hash
```

Comment:
Would you change "to" into "in order to"?  It took me three tries reading this sentence to understand that the phrase "needs to create" means "is required to have in order to create" instead of "should generate"